lib/iris/__init__.py

```diff
@@ -113,6 +113,7 @@ def callback(cube, field, filename):
 from iris._deprecation import IrisDeprecation, warn_deprecated
 import iris.fileformats
 import iris.io
+import iris.options
```

Comment:
@dkillick It's a moot point at the moment since `iris.options` doesn't actually exist :stuck_out_tongue_winking_eye: